core/src/components/item-divider/item-divider.md.scss

```diff
@@ -44,7 +44,10 @@
 ::slotted(ion-icon) {
   color: $item-md-icon-slot-color;
 
-  font-size: $item-md-icon-slot-font-size;
+  // The font size should be 24px which is
+  // calculated in relation to the host's font
+  // size of 14px. e.g. 24 / 14 = 1.714
+  font-size: 1.71em;
```

Comment:
Why was `1.71em` used instead of `1.714em`? Is using `1.71em` causing the screenshots to be updated?

Reply:
No legitimate reason. I just have been rounded them to 2 decimal places.

Reply:
Updated in https://github.com/ionic-team/ionic-framework/pull/27989/commits/9cb4f2739103706f12449dc450c36dc5f8e26db3